pkg/query-service/app/clickhouseReader/reader.go

```diff
@@ -3434,36 +3434,81 @@ func (r *ClickHouseReader) GetLogFields(ctx context.Context) (*model.GetFieldsRe
 func extractSelectedAndInterestingFields(tableStatement string, fieldType string, fields *[]model.LogField, response *model.GetFieldsResponse) {
 	for _, field := range *fields {
 		field.Type = fieldType
-		if isSelectedField(tableStatement, field.Name) {
+		// all static fields are assumed to be selected as we don't allow changing them
```

Comment:
nit: You probably want to update the references, such as "interesting" here https://github.com/SigNoz/signoz/blob/ec7c99dd26566b5b161a1dedab104e23485dff36/pkg/query-service/constants/constants.go#L200-L226

Reply:
Have made the change